Wikipedia/assets/preview.js

```diff
@@ -1745,8 +1787,7 @@ var IMAGE_LOADED_CLASS = 'pagelib_lazy_load_image_loaded'; // Download completed
 
 // Attributes copied from images to placeholders via data-* attributes for later restoration. The
 // image's classes and dimensions are also set on the placeholder.
-// The 3 data-* items are used by iOS.
-var COPY_ATTRIBUTES = ['class', 'style', 'src', 'srcset', 'width', 'height', 'alt', 'data-file-width', 'data-file-height', 'data-image-gallery'];
```

Comment:
These are the bits I accidentally merged too soon - they are required for this PR, but will be merged once we update pagelib version once these bits are merged there.